pkg/deploy/generator/scripts/gatewayVMSS.sh

```diff
@@ -1,12 +1,19 @@
+# We need to manually set PasswordAuthentication to true in order for the VMSS Access JIT to work
+echo "setting ssh password authentication"
+sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config
+
```

Comment:
We need to reload the SSH config or this won't take effect until the reboot.

Reply:
Indeed, thanks!